README.md

```diff
@@ -96,6 +93,20 @@ Recommended install is through fluentd's native OS based package installs:
 | MacOS/Darwin  | DMG             | https://docs.fluentd.org/installation/install-by-dmg |
 | Windows       | MSI             | https://docs.fluentd.org/installation/install-by-msi |
 
+__[EXPERIMENTAL]__ Alternatively, it's also possible to use the shell script to install Fluentd as service (experimental). The script installs td-agent 4 and requires sudo rights. 
```

Comment:
If we provide a script for td-agent can we provide a script that does the equivalent for user only installs w/ the zip installer? How much effort would this be to modify?

Reply:
It should be relatively easy (linux), the only issue related to this change is that it would require updating the script  every time we update the zip file name because the zip file name contains version number  (e.g fluentd-1.11.0-linux-x86_64.tar.gz, please correct me If I'm wrong). When I think about it the same issue  applies to all README.md (all the zip references: file name, path, etc). Maybe we can drop the version from the zip file name and add README that contains the version info? BTW In general more functionality should be added to the script as we progress. I’ll try to implement the suggestion change soon but it might take a couple of days before I’m able to go back to the log-analytics.

Reply:
@peters95, "User install" implemented however might be still shaky and might need more testing, commit:
https://github.com/jfrog/log-analytics/pull/63/commits/53cf271865a46038a7165c110c9e111bb6720493

Hopefully will have a chance to work on it some more soon. Still not ready to merge (more testing/bug fixing needed).